hm-gtm.php

```diff
@@ -40,37 +41,23 @@ function tag( $echo = true ) {
 				j.async=true;j.src=\'//www.googletagmanager.com/gtm.js?id=\'+i+dl;f.parentNode.insertBefore(j,f);
 			})(window,document,\'script\',\'dataLayer\',\'%1$s\');</script>
 			<!-- End Google Tag Manager -->
-			',
-			esc_attr( $id ),
-			$data_layer
-		);
-		$data_layer = '';
+			';
+
+	echo Plugin::data_layer();
+
+	if ( $id ) {
+
+		$output .= sprintf( $tag, esc_attr( $id ) );
 	}
 
-	if ( is_multisite() ) {
-
-		$id = get_site_option( 'hm_network_gtm_id' );
-
-		if ( $id ) {
-			$tag .= sprintf( '
-				%2$s
-				<!-- Google Tag Manager - Network Wide -->
-				<noscript><iframe src="//www.googletagmanager.com/ns.html?id=%1$s" height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
-				<script>(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({\'gtm.start\': new Date().getTime(),event:\'gtm.js\'});
-					var f=d.getElementsByTagName(s)[0], j=d.createElement(s), dl=l!=\'dataLayer\'?\'&l=\'+l:\'\';
-					j.async=true;j.src=\'//www.googletagmanager.com/gtm.js?id=\'+i+dl;f.parentNode.insertBefore(j,f);
-				})(window,document,\'script\',\'dataLayer\',\'%1$s\');</script>
-				<!-- End Google Tag Manager -->
-				',
-				esc_attr( $id ),
-				$data_layer
-			);
-		}
+	if ( is_multisite() && $network_id ) {
```

Comment:
@sanchothefat I'm wondering if we should remove `is_multisite()` here because this option can be set only in the network area thus it's possible only for multisite installation.